scout/server/blueprints/variants/views.py

```diff
@@ -297,12 +303,14 @@ def sv_variants(institute_id, case_name):
     # Populate chromosome select choices
     controllers.populate_chrom_choices(form, case_obj)
 
-    genome_build = "38" if "38" in str(case_obj.get("genome_build")) else "37"
+    genome_build = "38" if "38" in str(case_obj.get("genome_build", "38")) else "37"
```

Comment:
Huh? Use 38 if the key is missing, use 37 if the key is any false value. I feel this warrants a bit of explanation, as it is done differently in other places..

Reply:
Line 300 on main branch is going to make the app crash for cases without genome build, but we didn't experience any crash at this level so I'm assuming that it is safe to say that all cases that are currently used in Scout have a build. 
For the cases with build "37" or those with "GRCh37" (super old, or in some other facilities) genome build will be set to "37" so it is fine. It's a way of projecting us towards the future, i.e. build 38 ?

Reply:
Ok perhaps line 300 is not causing a crash since you'd have "None", I can put a 37 there, just to be on the safe side, but I don't think we have any of those cases without the build any more

Reply:
Or I can keep line 300 as it is and use it in the other places

Reply:
Yes, either of those should be fine! I kind of like the first option of making it explicit. It is not super obvious in Python that you will get a string back from `str(None)` - but not a 0 from `int(None)`. But make the default either "37" or "38". This monkey brain is confused and needs to find meaning in the choice if the possible default outcomes differ. 😁